be/src/exprs/agg/nullable_aggregate.h

```diff
@@ -803,7 +803,7 @@ class NullableAggregateFunctionVariadic final
         if (has_nullable_column) {
             if (has_null) {
                 for (size_t i = 0; i < chunk_size; ++i) {
-                    if (!null_data_result[i] & !selection[i]) {
+                    if (!null_data_result[i] && !selection[i]) {
```

Comment:
This way of writing may be for performance.

Reply:
I check that if there is `-O1` or above, `&` and `&&` will produce the same assembly instruction: https://godbolt.org/z/W6WGjsz7d

Are there other conditions?